### PR TITLE
Add cname allowlist

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/FiltersRegistry/pull/828

### CODE REVIEW

**Review by piquark6046 — COMMENTED** [review]
I think that opening related PR in AdguardTeam/FiltersRegistry is necessary.

**Comment by zloyden** [comment]
Yeah, I'll open it.

**Review by zloyden — COMMENTED** [review]
Done. @jellizaveta please add this `https://github.com/AdguardTeam/FiltersRegistry/pull/828` to the description.

File: SpywareFilter/sections/cname_allowlist.txt [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+! Good:
+!       - ||example.org^$badfilter
+!       - @@||example.org^
+! Bad:
+!       - @@||example.org^$stealth (should be in AdGuard Base - allowlist_stealth.txt)
+!
+!
+! Used in shopping emails
+||data.digital.costco.ca^$badfilter
+||data.digital.costco.com^$badfilter
```
Comment (Alex-302) [inline SpywareFilter/sections/cname_allowlist.txt:14]: Эти правила не помогут. Проверь в DNS.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: SpywareFilter/sections/cname_allowlist.txt (hunk 1 above)
Comment (jellizaveta) [reply SpywareFilter/sections/cname_allowlist.txt:14]: В DNS отдельно исключила

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**